jupyter-docker/Dockerfile

```diff
@@ -11,13 +11,14 @@ USER root
 ENV DEBIAN_FRONTEND noninteractive
 ENV DEBIAN_REPO http://cdn-fastly.deb.debian.org
 
-RUN echo "deb $DEBIAN_REPO/debian jessie main\ndeb $DEBIAN_REPO/debian-security jessie/updates main" > /etc/apt/sources.list \
+RUN echo "deb $DEBIAN_REPO/debian stretch main\ndeb $DEBIAN_REPO/debian-security stretch/updates main" > /etc/apt/sources.list \
  && apt-get update && apt-get -yq dist-upgrade \
  && apt-get install -yq --no-install-recommends \
     nano \
+    wget \
+    gnupg \
     #note ca-certificates has to be before wget and curl
```

Comment:
Minor, remove this comment if it's not true :)

Reply:
ah thanks